functions/artsy/models/podcast.js

```diff
@@ -0,0 +1,88 @@
+var Q = require('q');
+var feedparser = require('feedparser-promised');
+var _ = require('underscore');
+_.mixin(require('underscore.string').exports());
+
+function Podcast() {
+  this.rssUrl = 'https://feeds.soundcloud.com/users/soundcloud:users:211089382/sounds.rss';
```

Comment:
I was going to suggest make this an `env` but I guess there is no point if we lose `env` everytime we deploy.